src/lib/structures/Command.ts

```diff
@@ -88,7 +88,7 @@ export class Command<PreParseReturn = Args, O extends Command.Options = Command.
 		this.fullCategory = options.fullCategory ?? this.location.directories;
 		this.typing = options.typing ?? true;
 
-		this.lexer = new Lexure.Lexer({
+		this.lexer = new Lexer({
```

Comment:
Or was it this that needed a default in the constructor / initial value? I forgot which nit and I can't find it on the review.

Reply:
This is one that needed the default and I can't do that. [This conversation](https://github.com/sapphiredev/framework/pull/468#discussion_r932626374) explains it.

Reply:
@samfundev Wait for next result and update to that.
https://github.com/sapphiredev/utilities/pull/404/files#diff-63b98d6beb14a27a657df5479d7a87f5136e762688cc29c9f4543a96ac8a492aR64

Reply:
wrong reply but this is for that map err bs